### PR TITLE
Add update avatar function

### PR DESCRIPTION
**Pull request by DnKler**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -67,11 +69,11 @@ def register():
     return render_template('register.html')
 
 
-@app.route('/cabinet')
+@app.route('/cabinet', methods=['GET', 'POST'])
 @login_required
 def cabinet():
     if request.method == "POST":
-        return wl_search(db)
```
Comment (anya768) [inline main.py:74]: Don't understand where the search function goes. Does id search work?

File: src/upload.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    return '.' in filename and \
+           filename.rsplit('.', 1)[1].lower() in ALLOWED_EXTENSIONS
+
+
+def update_avatar(db, username):
+    if 'file' not in request.files:
+        flash('No file part')
+    file = request.files['file']
+    if file.filename == '':
+        flash('No selected file')
```
Comment (anya768) [inline src/upload.py:19]: I'll change here some flash things if you don't mind)

**Review by anya768 — APPROVED** [review]

File: main.py (hunk 1 above)
Comment (DnKler) [reply main.py:74]: wl_search is located in search.py . It gets id from request.form["search"], I hope...)

File: src/upload.py (hunk 2 above)
Comment (DnKler) [reply src/upload.py:19]: Yes, thank you!

**Pull request merged**